src/modules/generators/checkerboard.rs

```diff
@@ -0,0 +1,87 @@
+// Copyright 2013 The Noise-rs Developers.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use num::{Float, One};
+use math;
+use Module;
+use math::{Point2, Point3, Point4};
+
+#[derive(Clone,Copy,Debug)]
```

Comment:
Add spacing between the commas. Could you ensure that the other modules have the same derives added to them?